pkg/printers/internalversion/describe.go

```diff
@@ -635,11 +635,13 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 		}
 		describeContainers("Containers", pod.Spec.Containers, pod.Status.ContainerStatuses, EnvValueRetriever(pod), w, "")
 		if len(pod.Status.Conditions) > 0 {
-			w.Write(LEVEL_0, "Conditions:\n  Type\tStatus\n")
+			w.Write(LEVEL_0, "Conditions:\n  Type\tStatus\tReason\n")
+			w.Write(LEVEL_1, "----\t------\t------\n")
 			for _, c := range pod.Status.Conditions {
-				w.Write(LEVEL_1, "%v \t%v \n",
-					c.Type,
-					c.Status)
+				if len(c.Reason) == 0 {
+					c.Reason = "<none>"
+				}
+				w.Write(LEVEL_1, "%v \t%v\t%v\n", c.Type, c.Status, c.Reason)
```

Comment:
Turn those entire for-loops into helper methods instead of copy&pasting it over and over again.